tutorial/installing-appliance.md

```diff
@@ -41,6 +41,23 @@ The Anbox Cloud Appliance is available through the AWS Marketplace and as a snap
 
 Installing the Anbox Cloud Appliance through the AWS Marketplace simplifies the installation and deployment process and allows billing to be handled directly through AWS.
 
+#### Before you start
+
+Deploying the Anbox Cloud Appliance requires some familiarity with AWS. In particular, you should be familiar with:
+
+- Amazon Elastic Compute Cloud (Amazon EC2), for basic EC2 configuration
+- Amazon Elastic Block Storage (Amazon EBS), for configuring the EC2 instance storage and managing encryption
+- Amazon Virtual Private Cloud (Amazon VPC), for configuring an internet facing subnet and a security group
+
+The appliance uses the following billable services by AWS:
+
+- EC2 and Marketplace appliance (see the AWS Marketplace product page for costs)
+- Network egress
+
+You do not need to request an increase in limits for your AWS account for this deployment.
+
+The entire deployment process will take 10-15 minutes, depending on the selected hardware and the network conditions.
+
```

Comment:
Please move this whole section down to line 57/74, so that it's basically a "step 0" in the instructions.